doc/tutorial/Client.lhs

```diff
@@ -1,12 +1,11 @@
 # Querying an API
 
-While defining handlers that serve an API has a lot to it, querying an API is simpler: we do not care about what happens inside the webserver, we just need to know how to talk to it and get a response back. Except that we usually have to write the querying functions by hand because the structure of the API isn't a first class citizen and can't be inspected to generate a bunch of client-side functions.
+While defining handlers that [serve an API](Server.lhs) has a lot to it, querying an API is simpler: we do not care about what happens inside the webserver, we just need to know how to talk to it and get a response back. Except that we usually have to write the querying functions by hand because the structure of the API isn't a first class citizen and can't be inspected to generate a bunch of client-side functions.
```

Comment:
Hmm yes. To me it's not so much wrong but colloquial / chatty in style. Happy to change it though.

How about _That said, we usually have to write.._ (it might not be the _only_ exception), and perhaps changing _a bunch of_ to just _the_?